corehq/apps/domain/deletion.py

```diff
@@ -285,9 +285,6 @@ def _delete_demo_user_restores(domain_name):
     ModelDeletion('sms', 'MobileBackendInvitation', 'domain'),
     CustomDeletion('sms', _delete_domain_backends, ['SQLMobileBackend']),
     CustomDeletion('users', _delete_web_user_membership, []),
-    ModelDeletion('users', 'SQLUserRole', 'domain', [
```

Comment:
Just curious, why did this need to move?

Reply:
It has to come after `_terminate_subscriptions` otherwise the SQL roles get re-created when the Couch roles are deactivated.